Package.swift

```diff
@@ -0,0 +1,86 @@
+// swift-tools-version:5.4.0
+
+import PackageDescription
+
+let package = Package(
+    name: "Ably",
+    platforms: [
+        .macOS(.v10_11),
```

Comment:
I'm happy with bumping minimum version of macOS to 10.11. @QuintinWillison , do you forsee any issues with this?